UILibraries/ImageSlider/ImageSlider/Classes/Core/ImageSliderView.swift

```diff
@@ -45,9 +45,7 @@ public final class ImageSliderView: UIView {
         return collection
     }()
     
-    private lazy var pager: ImageSliderPagerView = {
-        return ImageSliderPagerView(frame: .zero)
-    }()
+    private lazy var pager = ImageSliderPagerView(frame: .zero)
```

Comment:
simply just use `ImageSliderPagerView()` in this way the frame will be `zero`.

Reply:
Sure, it's just an existing code.